Makefile

```diff
@@ -0,0 +1,76 @@
+run:
+	@echo "*********************************\n"
+	@echo "Lifting up the whole environment!\n"
+	@echo "*********************************\n"
+	@docker-compose up
+
+run-db:
+	@echo "************************\n"
+	@echo "Lifting up the database!\n"
+	@echo "************************\n"
+	@docker-compose up falko-database
+
+run-api:
+	@echo "*************************\n"
+	@echo "Lifting up the api!\n"
+	@echo "*************************\n"
+	@docker-compose up falko-server
+
+console:
+	@echo "*************************\n"
+	@echo "Entering the rails console!\n"
+	@echo "*************************\n"
+	@docker-compose falko-server rails console
+
+create-db:
+	@echo "*************************\n"
+	@echo "Creating rails database!\n"
+	@echo "*************************\n"
+	@docker-compose exec falko-server bundle exec rails db:create
+
+seed:
+	@echo "*************************\n"
+	@echo "Executing database seed scripts\n"
+	@echo "*************************\n"
+	@docker-compose exec falko-server bundle exec rails db:seed
+
+down:
+	@echo "******************************\n"
+	@echo "Dropping down the environment!\n"
+	@echo "******************************\n"
+	@docker-compose down
+
+ps:
+	@echo "************************\n"
+	@echo "Listing running services\n"
+	@echo "************************\n"
+	@docker-compose ps
+
+migrate:
+	@echo "******************\n"
+	@echo "Migrating database\n"
+	@echo "******************\n"
+	@docker-compose exec falko-server bundle exec rails db:migrate
+
+test:
+	@echo "***************************\n"
+	@echo "Executing all api tests\n"
+	@echo "***************************\n"
+	@docker-compose exec falko-server bundle exec rails test
+
+rm-network:
+	@echo "**********************************\n"
+	@echo "Removing all networks!\n"
+	@echo "**********************************\n"
+	@docker network rm $(sudo docker network ls -q)
+
+rm-volume:
+	@echo "*********************\n"
+	@echo "Removing all volumes!\n"
+	@echo "*********************\n"
+	@docker volume rm $(sudo docker volume ls -q)
+
+.PHONY: no_targets__ list
```

Comment:
Adicione ao menos um comentário descrevendo essa função

Reply:
Esse target `.PHONY` não bem uma função, é apenas uma notação pra dizer que as variaveis de ambiente usadas no comando do target `list:` não são arquivos literais. [Ref](https://stackoverflow.com/questions/2145590/what-is-the-purpose-of-phony-in-a-makefile)